src/OrchardCore.Modules/OrchardCore.Indexing/Migrations.cs

```diff
@@ -19,6 +27,69 @@ await SchemaBuilder.AlterTableAsync(nameof(IndexingTask), table => table
             .CreateIndex("IDX_IndexingTask_ContentItemId", "ContentItemId")
         );
 
-        return 1;
+        PopulateIndexingTaskTableWithContentItems();
+
+        return 2;
+    }
+
+#pragma warning disable CA1822 // Mark members as static
+    public int UpdateFrom1()
+#pragma warning restore CA1822 // Mark members as static
+    {
+        PopulateIndexingTaskTableWithContentItems();
+        return 2;
+    }
+
+    private void PopulateIndexingTaskTableWithContentItems()
+    {
+        ShellScope.AddDeferredTask(async scope =>
+        {
+            var clock = scope.ServiceProvider.GetRequiredService<IClock>();
+            var session = scope.ServiceProvider.GetRequiredService<ISession>();
+            var dbConnectionAccessor = scope.ServiceProvider.GetRequiredService<IDbConnectionAccessor>();
+            var logger = scope.ServiceProvider.GetRequiredService<ILogger<Migrations>>();
+            var tablePrefix = session.Store.Configuration.TablePrefix;
+
+            logger.LogDebug("Updating IndexingTask records");
+
+            await using var connection = dbConnectionAccessor.CreateConnection();
+            await connection.OpenAsync();
+            var dialect = session.Store.Configuration.SqlDialect;
+
+            try
+            {
+                var schema = session.Store.Configuration.Schema;
+                var indexingTableName = dialect.QuoteForTableName($"{tablePrefix}{nameof(IndexingTask)}", schema);
+
+                var contentItemIdColumnName = dialect.QuoteForColumnName(nameof(IndexingTask.ContentItemId));
+                var createdUtcColumnName = dialect.QuoteForColumnName(nameof(IndexingTask.CreatedUtc));
+                var typeColumnName = dialect.QuoteForColumnName(nameof(IndexingTask.Type));
+
+                var indexingSelectorBuildr = dialect.CreateBuilder(tablePrefix);
+                indexingSelectorBuildr.Select();
+                indexingSelectorBuildr.Selector(nameof(IndexingTask), nameof(IndexingTask.ContentItemId), schema);
+                indexingSelectorBuildr.Table(nameof(IndexingTask), null, schema);
+
+                var sqlBuilder = dialect.CreateBuilder(tablePrefix);
+                sqlBuilder.Select();
+                sqlBuilder.Selector(nameof(ContentItemIndex), nameof(ContentItemIndex.ContentItemId), schema);
+                sqlBuilder.AddSelector(", @CreatedUtc, 0");
```

Comment:
Are you sure this `@` will work in all databases? Isn't there a helper in SqlBuilder? Or did you find the same thing somewhere else?